javase/src/main/java/com/google/zxing/client/j2se/translators/HtmlAssetTranslator.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.google.zxing;
+package com.google.zxing.client.j2se.translators;
```

Comment:
Hm, if it's just these two classes... maybe worthwhile. They are meant to be command-line utilities not so much called from other code. (The docs that show usage of them would have to reflect the new package)